guides-website/docs/typescript-style.md

```diff
@@ -554,3 +554,49 @@ function returnObject() {
 	};
 }
 ```
+
+## Import / Export
+
+- **Avoid export default**
+
+> We should avoid export default as the provide maintainability concerns. You can read more up about this topic in [Typescript Deep Dive - Avoid Export Default](https://basarat.gitbook.io/typescript/main-1/defaultisbad)
+
+Example: 
+
+```ts
+class Foo {
```

Comment:
When using examples that require multiple files, I recommend including a comment on the first line that names the file for clarity.

```ts
// foo.ts
class Foo {
}
export default Foo;
```

```ts
// bar.ts
import Foo from "./foo";
```